data/syntax-highlighting/vim/syntax/meson.vim

```diff
@@ -111,6 +111,7 @@ syn keyword mesonBuiltin
   \ message
   \ option
   \ project
+  \ response_file
```

Comment:
Wanna go make a similar PR for the vscode extension too?

Reply:
Where is it? There is a test that complains if vim syntax is missing a function, but I didn't saw something similar for vscode...

Reply:
https://github.com/mesonbuild/vscode-meson/blob/main/syntaxes/meson.tmLanguage.json#L67